perun-wui-core/src/main/java/cz/metacentrum/perun/wui/json/managers/UsersManager.java

```diff
@@ -252,6 +252,13 @@ public static Request validatePreferredEmailChange(int userId, String token, Jso
 
 	}
 
+	public static Request validateSSHKey(String sshKey, JsonEvents events) {
```

Comment:
To be consistent, add the javadoc here, please.
```suggestion
    /**
    * ...
    */
	public static Request validateSSHKey(String sshKey, JsonEvents events) {
```